packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -1049,63 +1048,57 @@ class DefaultTextStyleTransition extends AnimatedWidget {
 /// performance significantly in some cases and is therefore a good practice.
 ///
-/// {@tool snippet}
+/// {@tool dartpad --template=stateful_widget_material_ticker}
 ///
-/// This code defines a widget called `Spinner` that spins a green square
-/// continually. It is built with an [AnimatedBuilder] and makes use of the
-/// [child] feature to avoid having to rebuild the [Container] each time. The
-/// resulting animation is shown below the code.
+/// This code defines a widget that spins a green square continually. It is
+/// built with an [AnimatedBuilder] and makes use of the [child] feature to
+/// avoid having to rebuild the [Container] each time.
+///
+/// ```dart imports
+/// import 'dart:math' as math;
+/// ```
 ///
 /// ```dart
-/// class Spinner extends StatefulWidget {
-///   @override
-///   _SpinnerState createState() => _SpinnerState();
+/// AnimationController _controller;
+///
+/// @override
+/// void initState() {
+///   super.initState();
+///   _controller = AnimationController(
+///     duration: const Duration(seconds: 10),
+///     vsync: this,
+///   )..repeat();
 /// }
 ///
-/// class _SpinnerState extends State<Spinner> with SingleTickerProviderStateMixin {
-///   AnimationController _controller;
-///
-///   @override
-///   void initState() {
-///     super.initState();
-///     _controller = AnimationController(
-///       duration: const Duration(seconds: 10),
-///       vsync: this,
-///     )..repeat();
-///   }
-///
-///   @override
-///   void dispose() {
-///     _controller.dispose();
-///     super.dispose();
-///   }
+/// @override
+/// void dispose() {
+///   _controller.dispose();
+///   super.dispose();
+/// }
 ///
-///   @override
-///   Widget build(BuildContext context) {
-///     return AnimatedBuilder(
-///       animation: _controller,
-///       child: Container(
-///         width: 200.0,
-///         height: 200.0,
-///         color: Colors.green,
-///         child: const Center(
-///           child: Text('Wee'),
-///         ),
+/// @override
+/// Widget build(BuildContext context) {
+///   return AnimatedBuilder(
+///     animation: _controller,
+///     child: Container(
+///       width: 200.0,
+///       height: 200.0,
+///       color: Colors.green,
+///       child: const Center(
+///         child: Text('Whee!'),
 ///       ),
-///       builder: (BuildContext context, Widget child) {
-///         return Transform.rotate(
-///           angle: _controller.value * 2.0 * math.pi,
-///           child: child,
-///         );
-///       },
-///     );
-///   }
+///     ),
+///     builder: (BuildContext context, Widget child) {
+///       return Transform.rotate(
+///         angle: _controller.value * 2.0 * math.pi,
+///         child: child,
+///       );
+///     },
+///   );
 /// }
 /// ```
 /// {@end-tool}
 ///
-/// {@animation 300 300 https://flutter.github.io/assets-for-api-docs/assets/widgets/animated_builder.mp4}
```

Comment:
Why remove the animation?

Reply:
Oh, because it's a DartPad thing now and you can see it animate right there in the DartPad output?

Should we remove this from the assets-for-api-docs repo then (assuming its not referenced elesewhere)?

Reply:
Yes, that's why.  I'm reluctant to remove it from the assets-for-api-docs-repo, because there's an ugly secret that we need to resolve: the dartpad samples don't work offline in the Zeal/Dash docset. I've been meaning to look into how we can run the docs generation twice in CI, once with the dartpad config, and once with the old config, but I haven't gotten to it yet.

So, if I solve that problem, I might need all the animations again to embed in the offline docs.

Reply:
Should we leave a breadcrumb to that animation here (e.g. in a regular comment)?